src/python/foglamp/core/service_registry/service_registry.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""Core server module"""
+
+import time
+
+from aiohttp import web
+
+from foglamp.core.service_registry.instance import Service
+
+__author__ = "Amarendra Kumar Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+__start_time = time.time()
+
+
+async def ping(request):
+    since_started = time.time() - __start_time
+
+    return web.json_response({'uptime': since_started})
+
+async def register(request):
+    """
+    Register a service
+
+    :Example: curl -d '{"type": "Storage", "name": "Storage Services", "address": "127.0.0.1", "port": "8090"}'  -X POST  http://localhost:8082/foglamp/service
+    """
+
+    try:
+        data = await request.json()
+
+        service_name = data.get('name', None)
+        service_type = data.get('type', None)
+        service_address = data.get('address', None)
+        service_port = data.get('port', None)
+
+        if not (service_name or service_type or service_address or service_port):
```

Comment:
should we check with stack of validation error? and port should be checked for type int as well?